notebooks/official/generative_ai/text_embedding.ipynb

```diff
@@ -0,0 +1,1825 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    for i in range(record_count):
</pre><p>Not sure how easy it is due to the shape of the data, but this might be simpler with a list comprehension?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>